package.json

```diff
@@ -19,6 +19,8 @@
     "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@babel/preset-env": "^7.5.4",
     "@babel/register": "^7.4.4",
+    "@vue/test-utils": "^1.0.0-beta.29",
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
I know you removed it but it breaks vue-jest (https://github.com/vuejs/vue-jest/issues/160).

Reply:
Put it back if you need it